api/config/config_local.json

```diff
@@ -56,12 +56,6 @@
       "methodLogicalVersion": 3,
       "minExtractionScatterTasks": 20,
       "extractionScatterTasksPerSample": 4
-    },
-    "cdrv8plus": {
```

Comment:
confirmed that Local runs without this, and also submits the v7 extraction correctly